pkg/instance/instance.go

```diff
@@ -208,29 +208,29 @@ func (i *Instance) SetImageInstant(ctx context.Context, image string) error {
 }
 
 // SetCommand sets the command to run in the instance
-// This function can only be called when the instance is in state 'Preparing' or 'Committed'
+// This function can only be called when the instance is in state 'Preparing'
 func (i *Instance) SetCommand(command ...string) error {
-	if !i.IsInState(StatePreparing, StateCommitted) {
+	if !i.IsInState(StatePreparing) {
```

Comment:
Since we check only one state, can we just use `if i.state != StatePreparing` ? b/c `.IsInState` loops over all states